src/containers/ConnectorManagement.jsx

```diff
@@ -185,12 +187,27 @@ export default class ConnectorManagement extends Component {
       })
   }
 
-  updateAccount (connector, idx, values) {
+  async updateAccount (connector, idx, values) {
     const { t } = this.context
-    this._updateAccount(connector, idx, values)
-      .then(() => {
-        Notifier.info(t('account config success'))
-      })
+
+    const modifiedAccount = {
+      auth: {
+        login: values.login,
+        password: values.password
+      }
+    }
+    const folder = await cozy.client.files.statById(connector.accounts[idx].folderId, false, {limit: 1})
```

Comment:
Calls to cozy.client should be abstracted in store (here `CollectStore`), and never be done directly in components)